101-securitycenter-create-automation-for-all-recommendations/README.md

```diff
@@ -0,0 +1,51 @@
+# Sample name
```

Comment:
Use a proper title...

Also, check the pipeline test results, a few things need to be fixed there, then we can do a full review

Reply:
@bmoore-msft 
Fixed the issue with the read me.
Also Iv'e checked the piplilne and I v'e saw the following error:
##[error]        Using "[listCallbackURL(concat(" is not allowed - found on line: 80 for property: value

and also:
Output contains secret: triggerURI

Our API uses the the logic apps triggerURI as it's basis(it's required). Appreciate if you can advise what to do in this siutation.